### PR TITLE
User story 1 (2nd review)

### PR DESCRIPTION
**Pull request by j-eick**
- Mock-Data
  - Example running data can be found in src > components > lib > mock-data.json 
 
- Table + Form components are separated
  - for better overview
  - table component will be used again on future features (most likely)

- Styling
  - For both RunDataForm and Table components I didn't outsource the styling yet, to avoid having too many files with hardly any content. I will outsource the styling as soon as (1) the components are self-containing (2) there's more styling.

### CODE REVIEW

**Review by JessicaLoers — COMMENTED** [review]
Hi @j-eick 
This is a test in terms of the missing blue bobble  😁

File: src/components/RunDataForm/index.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import {useForm} from 'react-hook-form';
+import styled from 'styled-components';
```
Comment (JessicaLoers) [inline src/components/RunDataForm/index.js:2]: 👍

File: src/styles.js [hunk 2]
```diff
@@ -5,14 +5,18 @@ export const GlobalStyle = createGlobalStyle`
 	*::before,
 	*::after {
 		box-sizing: border-box;
+
```
Comment (JessicaLoers) [inline src/styles.js:8]: Please delete empty lines -  but you can do that in the next pull request 😉

**Review by JessicaLoers — APPROVED** [review]
**Hi @j-eick**  ✌️
well done and you're good to go for the QA.

**Congrats to your first approved PR** 🎉

File: src/styles.js (hunk 2 above)
Comment (j-eick) [reply src/styles.js:8]: Oh boy... those lines are so easy to overlook. 🙈 Thanks for pointing it out. 👍🏼

**Pull request closed without merging**